Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1399,7 +1404,7 @@ open class NavigationMapView: UIView {
                     self.travelAlongRouteLine(to: location.coordinate)
                 }
             default:
-                if self.simulatesLocation, let locationProvider = self.mapView.location.locationProvider {
+                if self.simulatesLocation && self.inActiveNavigation, let locationProvider = self.mapView.location.locationProvider {
```

Comment:
During the free drive, we don't send the location to trigger the `locationManager` update because the map puck is already listening to the location update.